### DISCUSSION TITLE
Error Domain=NSCocoaErrorDomain Code=3840

### DISCUSSION THREAD

**Discussion by icklaus** [codemagic-ci-cd/codemagic-docs#2106] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Unity

### Steps to reproduce

I am getting an error while build the ipa for distribution. what would be the possible cause?
- name: Build ipa for distribution
        script: |
            xcode-project build-ipa \
            --project "$UNITY_IOS_DIR/$XCODE_PROJECT" \
            --scheme "$XCODE_SCHEME"

### Expected results

ipa Build success

### Actual results

I am getting :
Error Domain=NSCocoaErrorDomain Code=3840 "JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0." UserInfo={NSDebugDescription=JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0., NSJSONSerializationErrorIndex=0}
2023-10-11 15:28:41.221 xcodebuild[3044:41117] Writing error result bundle to /var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/ResultBundle_2023-11-10_15-28-0041.xcresult
xcodebuild: error: Unable to read project 'Unity-iPhone.xcodeproj'.
        Reason: The project ‘Unity-iPhone’ is damaged and cannot be opened due to a parse error. Examine the project file for invalid edits or unresolved source control conflicts.

**Comment by himesh-cm** [comment]
@icklaus, apologies for the delayed reply. Could you please provide me with a build id so that I can investigate the issue further?

**Reply by himesh-cm** [reply]
Hi @icklaus, do you have any updates for me? Please confirm if you are still facing any issues.

**Comment by himesh-cm** [comment]
@icklaus, I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
